### PR TITLE
android fixes for player stat card

### PR DESCRIPTION
**Pull request by carteryu**
you'd no longer get the virtualized list warning on android.

also accidentally commited the fix for profile drawer refresh on this branch.. if pedro you already did that lmk and i can remove mine.

Screen Shot 2020-01-05 at 9 34 18 PM
Screen Shot 2020-01-05 at 9 34 40 PM

### CODE REVIEW

**Comment by rauchp** [comment]
Your fix for the profile view is way simpler, but mine is slightly faster since I remove the need for await’s in saving the new info (bc mine is backed my local- yours is backed by a refresh) I’ll let Cody be the judge.

**Comment by carteryu** [comment]
@rauchp no id say just go ahead with yours, we shouldnt need to refresh when the stuffs there locally. lll rm mine before merging this

**Comment by rauchp** [comment]
U sure? Yours is a lot less engineered than mine, which I like. If the load back to the drawer is fast then there’s a good case for both

**Comment by rauchp** [comment]
actually, if you weren’t working off my branch from this morning when I moved some shit to make profile view faster, your solution isn’t gonna work cuz I broke some shit w it originally

**Comment by carteryu** [comment]
> actually, if you weren’t working off my branch from this morning when I moved some shit to make profile view faster, your solution isn’t gonna work cuz I broke some shit w it originally

yea im sure
thx!

File: civilapp/src/current/common/PlayerStatCard.js [hunk 1]
```diff
@@ -55,24 +57,25 @@ class PlayerStatCard extends Component {
...
+        </View>
       </View>
     )
   }
+
+  renderStatBox = (tabNum, index) => {
+    const statTuple = this.statDict[tabNum * 4 + index]
+    const key = statTuple ? Object.keys(statTuple)[0] : ''
+    const val = statTuple ? statTuple[key] : ''
+    return <StatBox topText={key} bottomText={val}></StatBox>
```
Comment (EphemeralX) [inline civilapp/src/current/common/PlayerStatCard.js]: nit

File: civilapp/src/current/common/PlayerStatCard.js [hunk 2]
```diff
@@ -150,6 +153,7 @@ class PlayerStatCard extends Component {
           <View style={{ flexGrow: 1, alignItems: 'center', justifyContent: 'center' }}>
             <View style={styles.scrollableTabParent}>
               <ScrollableTabView
+                style={isIOS ? {} : { width: '100%' }}
```
Comment (EphemeralX) [inline civilapp/src/current/common/PlayerStatCard.js]: does setting width to 100% on both platforms mess it up?

File: civilapp/src/current/SportsTab.js [hunk 3]
```diff
@@ -524,6 +526,7 @@ class SportsTab extends React.Component {
 
         <View style={[styles.centeredView, { marginHorizontal: LeftwardMargin + 1.5 }]}>
           <ScrollableTabView
+            style={isIOS ? {} : { width: '100%' }}
```
Comment (EphemeralX) [inline civilapp/src/current/SportsTab.js]: does setting width to 100% on both platforms mess it up?

File: civilapp/src/current/RoomDetails.js [hunk 4]
```diff
@@ -583,7 +584,12 @@ class RoomDetails extends React.Component {
   render() {
     return (
       <RetroBackground>
-        <ScrollableTabView locked initialPage={0} page={this.state.tabIndex} renderTabBar={() => <CustomNoViewBar />}>
+        <ScrollableTabView
+          style={isIOS ? {} : { width: '100%' }}
```
Comment (EphemeralX) [inline civilapp/src/current/RoomDetails.js]: does setting width to 100% on both platforms mess it up?

File: civilapp/src/current/common/PlayerStatCard.js (hunk 2 above)
Comment (carteryu) [reply civilapp/src/current/common/PlayerStatCard.js]: yeah it does

**Pull request merged**